blocks/redirects-formatter/redirects-formatter.js

```diff
@@ -0,0 +1,175 @@
+  const checkboxSection = document.querySelector('.checkbox-container');
+
+  return urls.reduce((rdx, urlPair) => {
+    if (!locales.length) handler(NO_LOCALE_ERROR, checkboxSection);
+
+    locales.forEach((locale) => {
+      let from;
+      let to;
+      try {
+        from = new URL(urlPair[0]);
+      } catch (e) {
+        // eslint-disable-next-line no-console
+        handler(e.message, inputSection);
+        return;
+      }
+      try {
+        to = new URL(urlPair[1]);
+      } catch (e) {
+        // eslint-disable-next-line no-console
+        handler(e.message, inputSection);
+        return;
+      }
+      const fromPath = from.pathname.split('.html')[0];
+      rdx.push([`/${locale}${fromPath}`, `${to.origin}/${locale}${to.pathname}`]);
+    });
+    return rdx;
+  }, []);
+}
+
+export function stringifyListForExcel(urls) {
+  return urls.reduce((rdx, url) => {
+    // eslint-disable-next-line no-param-reassign
+    rdx += `${url[0]}\t${url[1]}\n`;
+    return rdx;
+  }, '');
+}
+
+export default async function init(el) {
+  const { createTag } = await import(`${getLibs()}/utils/utils.js`);
+  const xlPath = './locale-config.json';
+  const resp = await fetch(xlPath);
+  if (!resp.ok) return;
+  const { data } = await resp.json();
+
+  // Main container
+  const redirectsContainer = createTag('section', { class: 'redirects-container' });
+
+  // Header
+  const header = createTag('h1');
+  header.innerText = 'Redirect Formatting Tool';
+
+  // Error section
+  const errorSection = createTag('p', { class: 'error' });
+
+  // Checkboxes
+  const checkBoxesHeader = createTag('p', { class: 'cb-label' });
+  checkBoxesHeader.innerText = 'Select Locales';
+  const checkBoxes = await createLocaleCheckboxes(data);
+  const checkBoxesContainer = createTag('div', { class: 'checkbox-container' }, checkBoxes);
+  const selectAllCB = createTag('button', { class: 'select-all-cb' });
+  selectAllCB.innerText = SELECT_ALL_REGIONS;
+  const checkBoxesArea = createTag('section', { class: 'cb-area' }, [checkBoxesHeader, selectAllCB, checkBoxesContainer]);
+
+  // Text input area
+  const inputAreaContainer = createTag('section', { class: 'input-container' });
+  const textAreaInput = createTag('textarea', { class: 'redirects-text-area', id: 'redirects-input', name: 'redirects-input' });
+  const taiLabel = createTag('label', { class: 'io-label', for: 'redirects-input' });
+  taiLabel.innerText = 'Paste URLs here:';
+  const submitButton = createTag('button', { class: 'process-redirects' });
+  submitButton.innerText = 'Process Redirects';
+  inputAreaContainer.append(taiLabel, submitButton, textAreaInput);
+
+  // Text output Area
+  const outputAreaContainer = createTag('section', { class: 'output-container' });
+  const textAreaOutput = createTag('textarea', { class: 'redirects-text-area', id: 'redirects-output', name: 'redirects-output' });
+  textAreaOutput.setAttribute('readonly', 'true');
```

Comment:
Can't you add this attribute along with the others?
```suggestion
  const textAreaOutput = createTag('textarea', { class: 'redirects-text-area', id: 'redirects-output', name: 'redirects-output', readonly: true });
```